src/elements/content-preview/stories/ContentPreview.stories.js

```diff
@@ -1,54 +1,94 @@
 // @flow
 import * as React from 'react';
+import BoxAnnotations from 'box-annotations';
+import { button, text } from '@storybook/addon-knobs';
 import { IntlProvider } from 'react-intl';
 import ContentPreview from '../ContentPreview';
 import notes from './ContentPreview.notes.md';
 
-export const Preview = () => (
-    <IntlProvider locale="en">
-        <ContentPreview features={global.FEATURES} fileId={global.FILE_ID} hasHeader token={global.TOKEN} />
-    </IntlProvider>
-);
+// Used to re-render the story with props from the knobs
+let key = 0;
```

Comment:
Is there a way to do this without being declared in the global scope?

Reply:
There are ways but this is already just a tweak for storybook to be able to force-reRender the story when if you change the access token and file at the same time and the scope should only affect the story.

Reply:
If we want to force refresh when the token changes, it doesn't seem like we need the ContentPreview changes. Regarding the key, can we do something like this?

```
const fileId = text('File ID', global.FILE_ID);
const token = text('Access Token', global.TOKEN);

return (
        <IntlProvider locale="en" key={`${fileId}-${token}`>
            <ContentPreview
                features={global.FEATURES}
                fileId={fileId}
                hasHeader
                token={token}
            />
        </IntlProvider>
    );
```